nixos/modules/services/hardware/lirc.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lirc;
+in {
+
+  ###### interface
+
+  options = {
+    services.lirc = {
+
+      enable = mkEnableOption "LIRC daemon";
+
+      options = mkOption {
+        type = types.lines;
+        example = ''
+          [lircd]
+          nodaemon = False
+        '';
+        description = "LIRC default options (lirc_options.conf)";
+      };
+
+      config = mkOption {
```

Comment:
Where is ```config``` used?

Reply:
Thanks for review. I updated the description to clarify the option. The content is written into a file, which is passed to lircd as configuration.